src/app/Main/Body/QueryPage/Results/TableResults/TableResults-TableMainHeader.component.tsx

```diff
@@ -5,67 +5,52 @@ import {
     primaryColor,
 } from "../../../../../Shared/Style/Style-MainTheme.component";
 
+const spacerStyle = (isRow: boolean, colWidth: number): SerializedStyles => css`
+    width: ${isRow ? "40px" : `${colWidth}px`};
+    height: ${isRow ? "50px" : "40px"};
+`;
+
 const titleDivStyle = (
-    isTitle: boolean,
-    isRow: boolean
+    isRow: boolean,
+    isTitle: boolean
 ): SerializedStyles => css`
     display: ${isTitle ? "flex" : "none"};
-    align-items: ${isRow ? "flex-end" : "center"};
-    justify-content: ${isRow ? "center" : "flex-end"};
+    flex-direction: ${isRow ? "column" : "row"};
+    align-items: center;
     background-color: ${primaryColor};
 `;
 const tableTitleStyle = (
-    isTitle: boolean,
     isRow: boolean,
-    height: number,
-    width: number
+    isTitle: boolean
 ): SerializedStyles => css`
     display: ${isTitle ? "flex" : "none"};
-    width: ${isRow ? "inherit" : `${width}px`};
-    height: ${isRow ? `${height}px` : "inherit"};
-    min-height: max-content;
-    min-width: max-content;
-    margin: ${isRow ? "0.3em 0" : "0"};
-    padding: ${isRow ? "0 0.2em" : "0.2em 0"};
-    justify-content: center;
-    align-items: center;
-    font-weight: normal;
-    background-color: ${primaryColor};
+    margin: 0;
     color: ${onPrimaryColor};
     writing-mode: ${isRow ? "vertical-lr" : "none"};
     transform: ${isRow ? "rotate(180deg)" : "none"};
 `;
 
 export interface TableMainHeaderProps {
-    isTitle: boolean;
     isRow: boolean;
-    height: number;
-    width: number;
+    isRowAndCol: boolean;
     text: string;
+    isTitle: boolean;
 }
 
 /**
  * @desc Returns one main header to wrap the result table
- * @param {boolean} isTitle - true if row/column is selected
  * @param {boolean} isRow - true if row is selected
- * @param {number} height - height of the main header
- * @param {number} width - width of the main header

```

Comment:
up to interface and some params are missing

Reply:
Done.